Cargo.toml

```diff
@@ -27,6 +27,7 @@ ctrlc = "3.2.5"
 chrono = { version = "0.4.38", default-features = false }
 chrono-tz = "0.9"
 directories = "5"
+crossbeam = "0.8.4"
```

Comment:
Wait why do we depend directly on crossbeam now...?

Reply:
Oh I think that was an intermediate thing before I fixed ui_test